### DISCUSSION TITLE
detailled guide to use  if for people in dpi country

### DISCUSSION THREAD

**Discussion by padima2** [database64128/swgp-go#21] [Q&A]
my wg server on ubuntu wasworking well until the country decide to activate the dpi.now it s impossible to get wireguard get working.
i have windows and android in client side
can you provide a step by step guide so it can help people wich are in my case as well

thank you for your time

**Comment by database64128** [comment]
Hi, thank you for your interest in this project. Unfortunately, the answer to your question is no. Setting up WireGuard and swgp-go requires a basic understanding of networking, which tutorials and step-by-step guides cannot teach.
